src/VsixV3/ProjectTemplates1.x/Microsoft.VisualStudio.NetCore.ProjectTemplates.1.x.vsmanproj

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need a new vsmanproj? Why not MergeManifest into the ProjectTemplate.vsmanproj?